uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -593,6 +593,31 @@
       });
 
       describe('manager opens leave request popup in edit mode', function () {
+        describe('when leave request date has same date as period end date', function () {
+          beforeEach(function () {
+            var approvalStatus = optionGroupMock.specificValue('hrleaveandabsences_leave_request_status', 'value', '1');
```

Comment:
Because we now check if absence period was defined by leave request dates (and not actually checking if it was submitted successfully), we may not need to actually submit it. So this line may go.